### PR TITLE
Miscellaneous user improvements

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #23791

some things I noticed while running the code

### CODE REVIEW

**Comment by lindsayad** [comment]
Initially open pull requests as draft please

**Comment by GiudGiud** [comment]
Should be ready now

File: modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C [hunk 1]
```diff
@@ -137,9 +137,12 @@ INSFVRhieChowInterpolator::INSFVRhieChowInterpolator(const InputParameters & par
     if (blockIDs() != var.blockIDs())
       mooseError("Block restriction of interpolator user object '",
                  this->name(),
-                 "' doesn't match the block restriction of variable '",
+                 "' (",
+                 std::accumulate(blocks().begin(), blocks().end(), std::string{}),
```
Comment (roystgnr) [inline modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C:141]: Isn't this just going to concatenate block names without any separating whitespace or punctuation?

**Review by roystgnr — DISMISSED** [review]
AssumingIhavemisunderstoodandtheblocknameoutputwon'tlooklikethisIseenootherproblems.

File: modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C (hunk 1 above)
Comment (GiudGiud) [reply modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C:141]: cant remember lemme check

Comment (GiudGiud) [reply modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C:141]: good catch! not sure what I was thinking

**Review by roystgnr — DISMISSED** [review]
Looks good if CI agrees.  I worry about whether there might be more places that were depending on `stringify()` giving machine-friendly rather than user-friendly output.

**Comment by GiudGiud** [comment]
CI wont agree immediately ugh

but I still think this is a natural default for output, while a simple comma is not particularly default for the machine

**Comment by GiudGiud** [comment]
green green alas

**Review by roystgnr — APPROVED** [review]

**Pull request merged**